packages/web/config/utils.ts

```diff
@@ -58,10 +58,6 @@ export async function saveAssetImageToTokensDir(
     console.info("Skipping image download for test environment");
   }
 
-  if (fs.existsSync(filePath)) {
-    return null;
-  }
```

Comment:
I noticed my local dev server was using testnet token images (like the testnet axelar WETH image instead of ETH image) with this line enabled.

Reply:
https://linear.app/osmosis/issue/FE-688/change-eth-asset-image